build/cqs.tsv

```diff
@@ -178,10 +179,10 @@ com.datastax.cassandra:cassandra-driver-core	3.0.0	provided
 com.datastax.cassandra:cassandra-driver-mapping	3.0.0	provided
 javax.media:jai_codec	1.1.3	provided
 javax.media:jai_core	1.1.3	provided
-org.apache.accumulo:accumulo-server-base	1.6.2	provided
 org.apache.hadoop:hadoop-client	2.5.2	provided
 org.apache.hadoop:hadoop-common	2.5.2	provided
 org.apache.kafka:kafka_2.11	0.8.2.1	provided
+org.apache.kafka:kafka_2.11	0.9.0.1	provided
```

Comment:
@jnh5y actually, our existing CQ is for 0.8+, so I think we're good:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=9082